app/controllers/admin/conferences_controller.rb

```diff
@@ -82,7 +82,7 @@ def update
       short_title = @conference.short_title
       @conference.assign_attributes(conference_params)
       send_mail_on_conf_update = @conference.notify_on_dates_changed?
-      delete_event_schedules if @conference.start_hour_changed? || @conference.end_hour_changed?
+      delete_event_schedules if @conference.valid? && (@conference.start_hour_changed? || @conference.end_hour_changed?)
```

Comment:
Yes I tried that first,but the problem with this approach is:
> @conference.start_hour_changed? || @conference.end_hour_changed?

will not give you true as the conference is now updated.
Then I thought of using previous_changes()  but that too did not give the desired results.